tests/model_serving/model_runtime/PATTERNS.md

```
@@ -0,0 +1,140 @@
+assert response.get("outputs")
+output = response["outputs"][0]
+actual_data = output.get("data", [])
+assert actual_data
+assert isinstance(actual_data, list)
+```
+
+## Fixture Chain
+
+Tests follow a strict fixture chain with `indirect=True` parametrization:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)